src/style.css

```
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+}
+
+[type="search"]::-webkit-search-cancel-button,
+[type="search"]::-webkit-search-decoration {
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 30px 62px;
+  line-height: 22px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  text-shadow: 2px 10px 20px gray;
```

This would be more aligned with designs:
```suggestion
  text-shadow: 0 4px 3px rgba(61, 78, 97, 0.3);
```